eng/codeOptimization.targets

```diff
@@ -9,6 +9,10 @@
   <Target Name="SetApplyNgenOptimization"
           Condition="'$(IsEligibleForNgenOptimization)' == 'true'"
           BeforeTargets="CoreCompile">
+    <PropertyGroup>
+      <IbcOptimizationDataDir Condition="'$(OSGroup)' == 'Unix'">$(IbcOptimizationDataDir)$(LinuxCoreFxOptimizationDataPackageId)\</IbcOptimizationDataDir>
```

Comment:
You may want to consider more than just Unix. https://github.com/dotnet/corefx/blob/master/eng/configurations/osgroups.props